mwapi/errors.py

```diff
@@ -36,3 +48,38 @@ class LoginError(RuntimeError):
     @classmethod
     def from_doc(cls, doc):
         return cls(doc.get('result'))
+
+
+class RequestError(requests.exceptions.RequestException):
```

Comment:
Oh!  We'll have our own exceptions with these names in the next library that we use.  IMO, the exception names matching the exception thrown is the most important bit.  

Anyway, I just pushed a change that uses the "from e" syntax.  :)